skbeam/core/tests/test_utils.py

```diff
@@ -178,8 +187,8 @@ def test_grid3d():
     # slice tricks
     # this make a list of slices, the imaginary value in the
     # step is interpreted as meaning 'this many values'
-    slc = [slice(_min + (_max - _min)/(s * 2),
```

Comment:
Here is the [broken build](https://travis-ci.org/CJ-Wright/scikit-beam/builds/154831704)